### ISSUE TITLE
clarification on paper result

### ISSUE THREAD

**Issue by YuanTingHsieh** [MiuLab/SlotGated-SLU#4]
In your paper, the ATIS Dataset
The ["Atten Base Liu and Lane 2016"](https://arxiv.org/pdf/1609.01454.pdf)
is reported as 
F1 94.2, Intent Accuracy 91, Sentence Accuracy 78.9

while in their paper and [this new paper](https://cpb-us-w2.wpmucdn.com/campuspress.yale.edu/dist/7/677/files/2018/03/NAACL_websiteVersion-v3c51w.pdf)

is reported as 
F1 95.98 and Intent Accuracy 98.21

I wonder if your experiment setting is different from theirs?
Could you kindly explain this?

Thank you!

**Comment by zhhongzhi** [comment]
I noticed that this implement used 500 samples from training data as validation set.

**Comment by sz128** [comment]
Find solid baselines in https://github.com/sz128/slot_filling_and_intent_detection_of_SLU.
